frontend/packages/pipelines-plugin/src/components/catalog/hooks/useApiResponse.ts

```diff
@@ -20,7 +20,7 @@ const useApiResponse: UseApiResponse = (url: string, hasPermission: boolean) =>
           const json = await res.json();
           if (mounted) {
             setLoaded(true);
-            setResult(json.data);
+            setResult(json.data ? json.data : json.packages);
```

Comment:
It looks like that this `useApiResponse`  hook is tightly coupled to the TektonHub API and not used in your artifactHub hooks. So this change isn't needed?